app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,12 @@
+from app.players.player import Player
+
+
+class Dwarf(Player):
```

Comment:
You need to inherit from Player, ABC here, because if you are inheriting from Player only - you are required to implement  abstract methods here